corehq/apps/indicators/utils.py

```diff
@@ -1,10 +1,10 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
-from corehq.util.quickcache import quickcache
+from corehq.util.quickcache import skippable_quickcache
 from dimagi.utils.couch.database import get_db
 
 
-@quickcache([], timeout=60)
+@skippable_quickcache([], lambda: settings.UNIT_TESTING, timeout=60)
```

Comment:
I think there is a contradiction in what you just said and the code in this PR. You depend on caching to find errors in code when writing tests, but here you have disabled the cache for all tests, so they will not have that benefit. How do you decide which caching should be universally disabled while testing and which should always be enabled?